docs/reference/landscape.md

```diff
@@ -42,13 +42,21 @@ This section is the same one you'd see in other Landscape platforms, with some c
 
 All other data is ignored by the Agent and passed on verbatim to the Landscape client. You can see the reference for all the other keys in the [Landscape repository](https://github.com/canonical/landscape-client/blob/master/example.conf).
 
+The following keys are used by the Landscape client:
+
+- `account_name`: The Landscape account name this computer belongs to. For self-hosted Landscape accounts, the account name defaults to “standalone”.
+- `url`: The main URL for the Landscape Server to connect this client to. This defaults to the URL of your Landscape account followed by `/message-system`.
+- `ping_url`: The ping URL you want this client to report to. This defaults to the URL of your Landscape account followed by `/ping`.
+- `log_level`: This can be one of: "debug", "info", "warning", "error", "critical"
+- (Optional) `registration_key`: An optional account-wide key used to register new clients. There is no key defined by default, but one can be set in your Landscape account.
+
```

Comment:
I'm not sure we want to keep this duplicated info here. Our idea was to do like a layered documentation:
- Link to Landscape's docs for stuff that does not change
- Document what we add/change

You're the expert though; what do you think is best?

Reply:
I wouldn't duplicate. Instead, I'd link to the upstream and then document just the diff (clarifying that it's just the diff).

Reply:
Thanks both! @tmihoc I like that set up - I'll link it to Landscape docs instead